protobuf/payload.proto

```diff
@@ -64,14 +64,16 @@ message CallTx {
     bytes Address = 2 [(gogoproto.customtype) = "github.com/hyperledger/burrow/crypto.Address"];
     // The upper bound on the amount of gas (and therefore EVM execution steps) this CallTx may generate
     uint64 GasLimit = 3;
+    // The upper bound on the price per unit of gas
+    uint64 GasPrice = 4;
     // Fee to offer validators for processing transaction
-    uint64 Fee = 4;
+    uint64 Fee = 5;
     // EVM bytecode
-    bytes Data = 5 [(gogoproto.customtype) = "github.com/hyperledger/burrow/binary.HexBytes", (gogoproto.nullable) = false];
+    bytes Data = 6 [(gogoproto.customtype) = "github.com/hyperledger/burrow/binary.HexBytes", (gogoproto.nullable) = false];
```

Comment:
Don't change these numbers, this means the burrow-js can no longer talk to burrow